Gemfile.lock

```diff
@@ -61,19 +63,18 @@ GEM
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    railties (6.0.3.3)
-      actionpack (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
+    railties (6.1.3)
+      actionpack (= 6.1.3)
+      activesupport (= 6.1.3)
       method_source
       rake (>= 0.8.7)
-      thor (>= 0.20.3, < 2.0)
-    rake (13.0.1)
-    thor (1.0.1)
-    thread_safe (0.3.6)
-    tzinfo (1.2.7)
-      thread_safe (~> 0.1)
-    zeitwerk (2.4.0)
-    zxcvbn-ruby (1.1.0)
+      thor (~> 1.0)
+    rake (13.0.3)
+    thor (1.1.0)
+    tzinfo (2.0.4)
+      concurrent-ruby (~> 1.0)
+    zeitwerk (2.4.2)
+    zxcvbn-ruby (1.2.0)
```

Comment:
### [zxcvbn-ruby](https://github.com/envato/zxcvbn-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.1.0 → 1.2.0

[_[change-log](https://github.com/envato/zxcvbn-ruby/blob/HEAD/CHANGELOG.md), [source-code](https://github.com/envato/zxcvbn-ruby)_]





<details>
<summary>Commits</summary>

A change of **13** commits. See the full changes on [the compare page](https://github.com/envato/zxcvbn-ruby/compare/v1.1.0...v1.2.0).

These are the first 10 commits:
- (f2f4c08) [Use mini_racer instead of therubyracer](https://github.com/envato/zxcvbn-ruby/commit/f2f4c0825a983e4255db0095075bc1a397a57ba1)
- (44e1aa4) [Allow ruby3.x](https://github.com/envato/zxcvbn-ruby/commit/44e1aa4a61162eea6f0ff24096c6f8921f6cce17)
- (6173966) [Explicitly add 3.0 in travis](https://github.com/envato/zxcvbn-ruby/commit/61739664d9c577e9d0c84fe0ecc5967398607a95)
- (8ca10bf) [Remove accidental file](https://github.com/envato/zxcvbn-ruby/commit/8ca10bf2820d579c6c61360d2d93053c7ed7a9ac)
- (d3f4106) [Merge pull request #32 from RSO/ruby-3](https://github.com/envato/zxcvbn-ruby/commit/d3f410626aa2475ab1e1fae7386fbfc9aeb5f425)
- (2c65da7) [Merge pull request #33 from RSO/use-mini-racer](https://github.com/envato/zxcvbn-ruby/commit/2c65da7d9da8c2a0402a1e6a418739fe66644a6c)
- (f212079) [CI/Test with GitHub Actions](https://github.com/envato/zxcvbn-ruby/commit/f21207905fca740c1ea83909146f2517bdb760b7)
- (d7e5839) [Update CHANGELOG](https://github.com/envato/zxcvbn-ruby/commit/d7e5839c46645d1a1229abd72b285fd12642aa3b)
- (ad56662) [Fix up links](https://github.com/envato/zxcvbn-ruby/commit/ad566625e70c3015d4d37de1940e68dd70ea6a93)
- (2e636b5) [Use HEAD commit in URIs to make it easier to change default …](https://github.com/envato/zxcvbn-ruby/commit/2e636b594d5464bd563bf22514b7d7687bcc59ef)


</details>